e2e-samples/GitIntegration-ConnectAndUpdateFromGit.ps1

```diff
@@ -0,0 +1,156 @@
+# This sample script calls the Fabric API to programmatically connect to and update the workspace with content in Git.
+
+# For documentation, please see:
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/connect
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/initialize-connection
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/update-from-git
```

Comment:
Add also 'https://learn.microsoft.com/en-us/rest/api/fabric/core/long-running-operations' since you use it